fg/settings/env_secrets.py

```diff
@@ -0,0 +1,144 @@
+# This file, dummy_secrets, provides an example of how to configure
+# sregistry with your authentication secrets. Copy it to secrets.py and
+# configure the settings you need.
+
+# Secret Key
+# You must uncomment, and set SECRET_KEY to a secure random value
+# e.g. https://djskgen.herokuapp.com/
+
+from .config import getenv
```

Comment:
Did you mean for this file to replace `dummy_secrets.py`?

Reply:
No I don't

Reply:
okay I understand the difference now, we're good here. Could you please change the top of the file (the header mentions the other file, dummy_secrets.py) It would be good to provide a similar header to explain the purposes of this file.

Reply:
Got it, will do